### ISSUE TITLE
Missing category label

### ISSUE THREAD

**Issue by CelineZhou03** [gulvarol/grocerydataset#1]
I want to train in the VOC2007 dataset format, but your dataset is missing a category name. Does your dataset have no function in this area?

**Comment by gulvarol** [comment]
I am sorry I don't understand your question. The annotation file (https://github.com/gulvarol/grocerydataset#annotationtxt) has the 'brand' id which corresponds to the category. You can convert the information in any format you like.

**Comment by CelineZhou03** [comment]
Thank you for your reply. 
Each <b_i> in this annotation.txt has a value of zero. I want to turn it into a trademark name for each product. in other words，I want to convert annotation.txt annotation information into the format of the .xml file in the annotation inside the VOC2007 dataset. Because I want to use faster r-cnn to train your data set.

**Comment by gulvarol** [comment]
Zero is the 'other' class. Not all of the class annotations are zero.

**Comment by jha-prateek** [comment]
Hi, there's a lot of mis-classification in annotation file. 

For example look at line 164 in Annotations.txt 
File name: C2_P04_N3_S4_1
Cat Ids: ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '1', '1', '1', '1', '4', '4', '4', '4', '4', '4', '4', '10', '10', '10']

First shelf of planogram contains all Marlboro(cat id: 1) but in annotations.txt contains all 0.
![C2_P04_N3_S4_1](https://user-images.githubusercontent.com/21065903/81084252-400d8580-8f13-11ea-88e3-8ac380549d55.JPG)

**Comment by gulvarol** [comment]
There could be bounding boxes that are not annotated as one of the brand categories:

So among the annotated ones, there should be no wrong labels (they are all manually verified). Among the remaining ones (0: "other" class), there could be missing labels because not every bounding box is exhaustively annotated.

**Comment by jha-prateek** [comment]
So, how to figure out which of the images are properly annotated?
Because, I see lot of them are not.

**Comment by gulvarol** [comment]
As I said, the bounding boxes that are annotated as non-zero are manually verified. The 'other' category has some noise, but is largely correct. You can easily annotate more yourself either manually or semi-automatically, as the number of categories are very small.

**Comment by jha-prateek** [comment]
Ok, Thanks.

**Comment by sayakpaul** [comment]
How is the [csv file](https://github.com/gulvarol/grocerydataset/blob/master/annotations.csv) structured? Am I right to assume it has the following structure? 

![image](https://user-images.githubusercontent.com/22957388/104796186-f7b0a200-57d9-11eb-9978-188f7c136d91.png)

@gulvarol

**Comment by sayakpaul** [comment]
I guess this should be the case - 

![image](https://user-images.githubusercontent.com/22957388/104796209-42321e80-57da-11eb-88c4-e9ab5d32e9a6.png)


Could you confirm? @gulvarol

**Comment by gulvarol** [comment]
I have not created the csv file, it was [commited](https://github.com/gulvarol/grocerydataset/commit/991a12aad117503678acbff636657e39c1c459e4) by @revantt

**Comment by revantt** [comment]
> I guess this should be the case -
> 
> ![image](https://user-images.githubusercontent.com/22957388/104796209-42321e80-57da-11eb-88c4-e9ab5d32e9a6.png)
> 
> Could you confirm? @gulvarol

Hi @sayakpaul  the headers for this file is [Image_id,x_min,y_min,x_max,y_max,label]

**Comment by sayakpaul** [comment]
Yeah got it. Maybe updating the csv to include the headers would help reduce this.

**Comment by sainivedh19pt** [comment]
The labelling of classes in annotations.csv in totally wrong. Surprised by how none has raised it yet
